dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.org.eclipse.microprofile.reactive.streams.operators-1.0.feature

```diff
@@ -6,6 +6,6 @@ singleton=true
   com.ibm.websphere.appserver.org.reactivestreams.reactive-streams-1.0, \
   com.ibm.websphere.appserver.classloading-1.0
 -bundles=\
-  com.ibm.websphere.org.eclipse.microprofile.reactive.streams.operators.1.0; location:="dev/api/stable/,lib/"; mavenCoordinates="org.eclipse.microprofile.reactive.streams:microprofile-reactive-streams-operators:1.0-M1"
+  com.ibm.websphere.org.eclipse.microprofile.reactive.streams.operators.1.0; location:="dev/api/stable/,lib/"; mavenCoordinates="org.eclipse.microprofile.reactive.streams:microprofile-reactive-streams-operators:1.0-RC4"
```

Comment:
The maven coords should be `org.eclipse.microprofile.reactive-streams-operators:microprofile-reactive-streams-operators-api:1.0`

Although I'm not sure what uses this information

Reply:
Edit done.